### PR TITLE
make libri speech button open a new window

### PR DESCRIPTION
**Pull request by mikehenrty**

### CODE REVIEW

**Comment by mikehenrty** [comment]
quick and dirty. @Gregoor let me know if you have a better way of doing this.

File: web/src/components/pages/data/data.tsx [hunk 1]
```diff
@@ -224,9 +224,20 @@ class DataPage extends React.Component<Props, State> {
                 </div>
 
                 <div className="action">
-                  <CardAction onClick={() => this.showModalFor(dataset)}>
-                    <DownloadIcon />Download
-                  </CardAction>
+                  {dataset.name === 'LibriSpeech' ? (
```
Comment (Gregoor) [inline web/src/components/pages/data/data.tsx]: We could replace the LibriSpeech dataset-object's download property with an empty array, and check for that here.

File: web/src/components/pages/data/data.tsx [hunk 2]
```diff
@@ -224,9 +224,20 @@ class DataPage extends React.Component<Props, State> {
...
+                    <Button
+                      outline
+                      className="card-action"
+                      onClick={() => {
+                        window.open('http://www.openslr.org/12/', '_blank');
```
Comment (Gregoor) [inline web/src/components/pages/data/data.tsx]: and then use `dataset.url` here

**Comment by mikehenrty** [comment]
@Gregoor ah good call! I will make those changes before submitting. Thanks!

**Pull request merged**